Assignment2/NicolásOrtizSuaña.ttl

```diff
@@ -8,11 +8,11 @@
 +:Class01          class:includes :Sensor029;
 +                  class:includes :Computer101.
 +
-+:Measurement8401  measurement:hasTemperature :"29"^^xsd:integer;
-+                  measurement:atTime :"2010-06-12T 12:00:12"^^xsd:dateTime.
++:Measurement8401  measurement:hasTemperature "29"^^xsd integer;
```

Comment:
This is still wrong. Please check what other students have been doing in the assignments that I have already accepted.